reference/component-versions.md

```diff
@@ -5,6 +5,43 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.23.1
```

Comment:
Ignore this for the first round of review, this can be added after we create the release candidate